package.json

```diff
@@ -34,6 +34,7 @@
     "prettier": "prettier --write 'packages/**/*.{js,jsx,ts,tsx,css}'"
   },
   "devDependencies": {
+    "cross-env": "^5.1.3",
```

Comment:
Is `cross-env` needed?

Reply:
Sorry... This is my mistake. I fixed it.